.eslintrc.js

```diff
@@ -118,7 +118,7 @@ module.exports = {
         'no-shadow': 2,
 
         // Vue
-        'vue/multi-word-component-names': ['error', {
+        'vue/multi-word-component-names': ['off', {
```

Comment:
I found this rule to be particularly pesky, resulting in a lot of bloated names like `pages/AboutPage`. I petition we switch it off. Thoughts?

Reply:
I'm in favor of switching it off and simplifying the component names!

Reply:
I had this on as it's a vue standard practice, but I don't have any preference either way, so let's turn it off